### PR TITLE
Implemented changes to adapt routing key template to Sepia

### PR DESCRIPTION
**Pull request by jainadc9**
### Applicable Issues

Implemented changes to adapt routing key template to Sepia

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: JainadC <jainad.chinta@ericsson.com>

### CODE REVIEW

File: src/main/java/com/ericsson/eiffel/remrem/semantics/RoutingKeyFamilies.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    EiffelServiceStartedEvent(EiffelConstants.SERVICE),
+    EiffelServiceStoppedEvent(EiffelConstants.SERVICE),
+    EiffelAlertRaisedEvent(EiffelConstants.ALERT),
+    EiffelAlertCeasedEvent(EiffelConstants.ALERT),
+    EiffelIssueDefinedEvent(EiffelConstants.TEST);
+
+
+    private final String family;   // family
+
+	RoutingKeyFamilies(String family) {
```
Comment (z-sztrom) [inline src/main/java/com/ericsson/eiffel/remrem/semantics/RoutingKeyFamilies.java:58]: Misaligned indentation.

File: src/main/java/com/ericsson/eiffel/semantics/events/Event.java [hunk 2]
```diff
@@ -55,24 +55,7 @@ public Meta generateMeta(Meta meta) {
...
-     * @param eventType
-     *            Eiffel eventType
-     * @return type based on eventType (or) error will be thrown if unknown
-     *         eventType requested.
-     */
-    public String getTypeRoutingKey(String eventType) {
-        try {
-            return RoutingKeyTypes.valueOf(eventType).getType();
+            return RoutingKeyFamilies.valueOf(eventType).getFamily();
         } catch (Exception e) {
```
Comment (z-sztrom) [inline src/main/java/com/ericsson/eiffel/semantics/events/Event.java:59]: Do we really need to catch a general exception? What if NullPointerException, ArrayIndexOutOfBoundsException is thrown in valueOf() or getFamily()? Then we incorrectly report "Unknown event type...".

**Comment by e-backmark-ericsson** [comment]
@jainadc9 can you answer on the comments provided?

File: src/main/java/com/ericsson/eiffel/remrem/semantics/RoutingKeyFamilies.java (hunk 1 above)
Comment (jainadc9) [reply src/main/java/com/ericsson/eiffel/remrem/semantics/RoutingKeyFamilies.java]: Aligned it

File: src/main/java/com/ericsson/eiffel/semantics/events/Event.java (hunk 2 above)
Comment (jainadc9) [reply src/main/java/com/ericsson/eiffel/semantics/events/Event.java]: Resolved

Comment (z-sztrom) [reply src/main/java/com/ericsson/eiffel/semantics/events/Event.java]: We really should not catch general exception here. Catch only exceptions declared by method valueOf(), i.e. IllegalArgumentException and NullPointerException.

**Review by z-sztrom — APPROVED** [review]

**Pull request merged**